competition/competition_bundle/track1_evaluation/config.yaml

```diff
@@ -2,3 +2,4 @@ phase: track1
 eval_episodes: 50
 seed: 42
 scenarios: []
+bubble_env_evaluation_seeds: []
```

Comment:
If `bubble_env_evaluation_seeds` is an empty list, then the additional bubble_env will not be run during Codalab evaluation (i.e., not included in competition scoring and ranking). It only runs during local evaluation by participants as the default config sets `bubble_env_evaluation_seeds=[6]`.

Was this the intended outcome?